docs/admin/users/idp-sync.md

```diff
@@ -420,36 +420,25 @@ settings, a user's memberships will update when they log out and log back in.
    Depending on the OIDC provider, this claim may be called something else.
    Common names include `groups`, `memberOf`, and `roles`.
 
-1. Configure the Coder server to read groups from the claim name with the OIDC
-   organization field server flag:
-
-   ```sh
-   # as an environment variable
-   CODER_OIDC_ORGANIZATION_FIELD=groups
-   ```
-
 1. Fetch the corresponding organization IDs using the following endpoint:
 
    ```text
    https://[coder.example.com]/api/v2/organizations
    ```
 
-1. Set the following in your Coder server [configuration](../setup/index.md).
+1. As a Coder admin, go to **Settings** > **IdP organization sync**.
 
-   ```env
-   CODER_OIDC_ORGANIZATION_MAPPING='{"data-scientists":["d8d9daef-e273-49ff-a832-11fe2b2d4ab1",  "70be0908-61b5-4fb5-aba4-4dfb3a6c5787"]}'
-   ```
+1. In the **Organization sync field** text box, enter the organization claim,
+   then select **Save**.
 
-   > One claim value from your identity provider can be mapped to many
-   > organizations in Coder. The example above maps to two organizations in
-   > Coder.
+   If there are already users in the default organization, leave **Assign
```

Comment:
As I understand it, when Assign Default Organization is enabled, users will always be added to the default organization and when it is disabled users will always be removed from the default organization. I think it may be useful to let users know that this is always the case whether or not users are already in the default organization.